addons/account/models/account_move.py

```diff
@@ -288,6 +288,13 @@ def _get_default_invoice_incoterm(self):
     inalterable_hash = fields.Char(string="Inalterability Hash", readonly=True, copy=False)
     string_to_hash = fields.Char(compute='_compute_string_to_hash', readonly=True)
 
+    _sql_constraints = [
+        ('unique_name',
+         """EXCLUDE (name WITH =, journal_id WITH =, move_type WITH =)
```

Comment:
> remove move_type (il most countries, sequences must be unique per journal, whatever the move_type)

That is not even true in Belgium...

> add an Integer computed field `uniq_suffix`: that will avoid changing the latam behaviour: this field is computed to NULL for all localisations except for vendor bills in CL,AR where it can be set to `partner_id`.

That field already exists, but you didn't want it at all when refactoring the sequences for version 14.0...